index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link href="registration.css" rel="stylesheet" type="text/css"> </head>
+
+<body>
+	<form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" name="test-form">
+		<fieldset class="personalInfo">
+			<legend>Personal information</legend>
+			<div class="form-field"> Surname:
+				<input type="text" name="surname" required> </div>
+			<div class="form-field"> Name:
+				<input type="text" name="name" required> </div>
+			<div class="form-field"> How old are you?
+				<input type="number" name="age"  minlength="1" maxlength="100" placeholder="12" required> </div>
+			<div class="form-field"> Full date of birth?
+				<input type="date" name="birthdate" placeholder="mm/dd/yyyy" required> </div>
+			<div class="form-field"> I accept the terms of the agreement
+				<input type="checkbox" name="terms" id="terms-and-conditions" required>
+				<label for="terms-and-conditions"></label>
+			</div>
+		</fieldset>
+		<fieldset class="registration">
+			<legend>Registration</legend>
+			<div class="form-field"> E-mail:
+				<input type="email" name="email" placeholder="email@example.com" required> </div>
+			<div class="form-field"> Password:
+				<input type="password" name="password"  autocomplete="off" required> </div>
+		</fieldset>
+		<fieldset class="interestingFact">
```

Comment:
```suggestion
		</fieldset>

		<fieldset class="interestingFact">
```

Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements

Fix all such cases